pkg/testutils/validation.go

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
move this file to test/lib

Reply:
Moved to `test/lib/validation` because of the import cycle problem.